Properly list needed libs.
Taken from Cygwin's R-makeconf.patch.
Might be needed on other system as well to avoid underlinking.
--- orig/etc/Makeconf.in	2016-03-17 00:04:44.000000000 +0100
+++ new/etc/Makeconf.in	2016-06-26 17:38:05.398615000 +0200
@@ -123,7 +123,7 @@
 ALL_OBJCXXFLAGS = $(PKG_OBJCXXFLAGS) $(CXXPICFLAGS) $(SHLIB_CXXFLAGS) $(OBJCXXFLAGS)
 ALL_FFLAGS = $(R_XTRA_FFLAGS) $(PKG_FFLAGS) $(FPICFLAGS) $(SHLIB_FFLAGS) $(FFLAGS)
 ## LIBR here as a couple of packages use this without SHLIB_LINK
-ALL_LIBS = $(PKG_LIBS) $(SHLIB_LIBADD) $(LIBR)@DYLIB_UNDEFINED_ALLOWED_FALSE@ $(LIBINTL)
+ALL_LIBS = $(PKG_LIBS) $(SHLIB_LIBADD) $(LIBR)@DYLIB_UNDEFINED_ALLOWED_FALSE@ $(LIBINTL) $(LIBS)
 
 .SUFFIXES:
 .SUFFIXES: .c .cc .cpp .d .f .f90 .f95 .m .mm .M .o
